packages/manager/modules/manager-pci-common/package.json

```diff
@@ -1,70 +1,90 @@
 {
   "name": "@ovh-ux/manager-pci-common",
   "version": "0.9.1",
-  "private": true,
-  "sideEffects": true,
+  "private": false,
+  "description": "Public Cloud Common components",
+  "homepage": "https://github.com/ovh/manager/blob/master/packages/manager/modules/manager-pci-common/README.md",
+  "bugs": {
+    "url": "https://github.com/ovh/manager/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/manager-pci-common"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
   "type": "module",
-  "main": "./src/index.ts",
+  "main": "dist/index.js",
+  "types": "dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
   "scripts": {
-    "build": "tsc --project tsconfig.build.json",
-    "dev": "tsc",
+    "build": "vite build",
+    "clean": "rimraf node_modules && rimraf dist && yarn cache clean",
+    "dev": "vite build",
```

Comment:
**question:** `vite ` itself isn't enough? I am not sure if HMR works in case of `vite build`.

Reply:
@anooparveti 
IMO, there is no point of having a dev server with HMR as this is a library, not an application.

The good way to develop a library in local is to associate a demo playground like StoryBook, as it has been done for MRC, to be the entry point of new changes.

So, meanwhile, we can keep like this or remove it as it's not manadatory. (StoryBook Will need a specific script)

What do you think?

Reply:
Are you planning to have a storybook or such documentation for components and other utils?

Reply:
@anooparveti Yes, I think this is a pre-requisite for developing and maintaining a library so we will need to implement it in a next future